hydradx/notebooks/Misc/load_omnipool_balances.ipynb

```diff
@@ -0,0 +1,468 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "7e3246da-b5f4-45e1-9aad-56772bff032b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\jepid\\PycharmProjects\\HydraDX-simulations\\hydradx\\notebooks\\Misc\n"
+     ]
+    }
+   ],
+   "source": [
+    "from matplotlib import pyplot as plt\n",
+    "import sys\n",
+    "import requests\n",
+    "import base64\n",
+    "import json\n",
+    "from pprint import pprint\n",
+    "from dotenv import load_dotenv\n",
+    "import os\n",
+    "import time\n",
+    "\n",
+    "sys.path.append('../..')\n",
+    "load_dotenv()\n",
+    "username = os.getenv('SQLPAD_USERNAME')\n",
+    "password = os.getenv('PASSWORD')\n",
+    "print(os.getcwd())\n",
+    "print(username, password)"
```

Comment:
I think we need to remove anything that prints the username or password in notebooks. This is setting a trap for users to check in their login credentials to github.